diplo.lua

```diff
@@ -0,0 +1,88 @@
+--[[
+diplo - Quick "Diplomacy"
+Without arguments:
+    Print list of civs the player civ has diplomatic relations to.
+    This list might not match the ingame civ list, but is what matters. Does not include 'no contact' civs, etc.
+    Also shows if relations match between player civ and other civ - a mismatch can potentially happen 'naturally', but is important when trying to make peace proper, as both civs need to be at peace with the other.
+With arguments:
+    diplo CIV_ID RELATION
+    Changes the relation to the civ identified by CIV_ID to the one specified in RELATION, making sure that this is mutual.
+    Relations: 0 = Peace; 1 = War; 3 = Alliance (seems to only apply to site govs...?)

```

Comment:
if we don't know the use of Alliance, I'd omit it from the docs (though maybe leave a comment about it in the code)

Reply:
Well, I'd have to do a lot more in-depth research to figure out how alliances and site govs interact with this, but since I removed the site gov listing/accessability it can go. War/peace is *supposed* to be at civ level, but I'm not sure, specially due to no-civ site gov tributes being possible...

Reply:
Removed mentions of alliance - further research needed to properly handle this relation.